services/core-web/webpack.config.ts

```diff
@@ -73,6 +73,9 @@ const commonConfig = merge([
       new HtmlWebpackPlugin({
         template: PATHS.template,
       }),
+      new webpack.ProvidePlugin({
```

Comment:
This exists for Core and Minespace.  It inserts the REQUEST_HEADER specific to each front end when utilizing it in the common workspace